fixtures/noop-132

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
```

Comment:
This does not test interleaving because `stderr` is not in-between two `stdout` calls.

Reply:
It was my mistake, I should have left it in-between and let the test fail. As I said in the comments I was testing various way to implement the test to be correct but I couldn't find one for all (more on this on the next conversation).

Reply:
Also I think it might be a good idea to do a more complex interleaving like stdout > stderr > stderr > stdout > stderr > stdout

Reply:
I don't think it needs to be more complex, if it works just once we should expect to interleave the rest of the outputs. Otherwise we know it's just concatenating the output strings.
From what I understand by the tests I'm doing is that we cannot guarantee the order of the interleaving.

Reply:
What do you think @sindresorhus?